### PR TITLE
Add a QR Code Page

### PR DESCRIPTION
**Pull request by JoeJanickiChSh**
Solves #52. Adds a page with a QR code which links to the registration page.

### CODE REVIEW

**Comment by msoucy** [comment]
I'd like to hold off on this until #55 is merged, so we can have QR codes for registering students, guardians, and mentors

**Comment by bot190** [comment]
Now that #55 is merged, can you add similar pages for the other registration pages? 
- [x] /register/mentor
- [x] /register/guardian

**Comment by JoeJanickiChSh** [comment]
All the pages should be added now.

**Review by msoucy — APPROVED** [review]

**Pull request merged**
